mappings/net/minecraft/client/gui/hud/chat/ChatHud.mapping

```diff
@@ -49,13 +50,18 @@ CLASS net/minecraft/unmapped/C_uedmlrsy net/minecraft/client/gui/hud/chat/ChatHu
 	METHOD m_gnxqpnev getMessageEndAt (DD)I
 		ARG 1 x
 		ARG 3 y
+	METHOD m_ieibwato addMessage (Lnet/minecraft/unmapped/C_skejqrts;)V
+		ARG 1 message
 	METHOD m_itcpovxa toChatLineX (D)D
 		ARG 1 x
+	METHOD m_iudfmgeq toChatState ()Lnet/minecraft/unmapped/C_uedmlrsy$C_frlehikz;
 	METHOD m_kyfuayab (ILnet/minecraft/unmapped/C_uedmlrsy$C_hrdqyvbn;)Z
 		ARG 2 message
 	METHOD m_lqhjyrrj createDelayedMessageDeletion (Lnet/minecraft/unmapped/C_shphkjbi;)Lnet/minecraft/unmapped/C_uedmlrsy$C_hrdqyvbn;
 		ARG 1 signature
 	METHOD m_maliiwol updateDeletionQueue ()V
+	METHOD m_mqmrcyqk addVisibleMessage (Lnet/minecraft/unmapped/C_skejqrts;)V
```

Comment:
```suggestion
	METHOD m_mqmrcyqk showMessage (Lnet/minecraft/unmapped/C_skejqrts;)V
```

maybe? i think addVisible is kinda naming on an impl detail

Reply:
I considered `showMessage` but I thought it was too different from `addMessage`, whereas it kind of does the same thing. `showMessage` imo implies rendering it, which admittedly it will but not _this frame_.

Reply:
For now won't resolve this conversation in order to discuss further if necessary.

Reply:
Maybe `queueMessage`?

Reply:
Might be better, though the visible/display/show element is missing, and it isn't too far off from `addMessage`. mojmap is `addMessageToDisplayQueue` so honestly in the right direction. Unsure what to do.

Reply:
`queueVisbileMessage`?

Reply:
I like that